Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-mocks (3.6.0)
+      rspec-support (~> 3.8.0)
+    rspec-mocks (3.8.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-support (3.6.0)
-    sequel (4.47.0)
+      rspec-support (~> 3.8.0)
+    rspec-support (3.8.0)
+    sequel (5.14.0)
```

Comment:
### [sequel](http://sequel.jeremyevans.net)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 4.47.0 → 5.14.0

[_[change-log](http://sequel.jeremyevans.net/rdoc/files/CHANGELOG.html), [source-code](https://github.com/jeremyevans/sequel)_]





<details>
<summary>Commits</summary>

A change of **830** commits. See the full changes on [the compare page](https://github.com/jeremyevans/sequel/compare/4.47.0...5.14.0).

These are the first 10 commits:
- (222ed49) [Bump copyright year](https://github.com/jeremyevans/sequel/commit/222ed490312f3edf829c9c0330021bf988d2ba22)
- (9a760ef) [Fix quotes mismatch in readme](https://github.com/jeremyevans/sequel/commit/9a760ef7cb94f11d0906426c50bdc24991a3b8b0)
- (2652844) [Bump version to 5.5.0](https://github.com/jeremyevans/sequel/commit/26528449ba9f7149c3d0e44c1db886936ae36e45)
- (8e3c9f4) [Respect identifier mangling rules when renaming columns on M…](https://github.com/jeremyevans/sequel/commit/8e3c9f40aa60cea65bef1381faa07fa5c22d9c5e)
- (8ddd3bc) [Fix typo in spec description](https://github.com/jeremyevans/sequel/commit/8ddd3bc9abd38f9caa1294044408bcc411864ca0)
- (99fea4e) [Shadowing outer local variable v](https://github.com/jeremyevans/sequel/commit/99fea4eb2e9f63895742175b308a056a3a4a1955)
- (296e27d) [Fix some additional verbose mode warnings](https://github.com/jeremyevans/sequel/commit/296e27d1352edb6894698b7d950ede8f2641f7ec)
- (c37775d) [Protect migration file loading with a mutex to not break whe…](https://github.com/jeremyevans/sequel/commit/c37775d4caf9cc8cae53ea2ad6b0882ca6f0018f)
- (4820929) [Freeze string valuse in hashes returned by Database#schema w…](https://github.com/jeremyevans/sequel/commit/48209297ea92b12c0cf427dd9c3098164f82af0a)
- (2f57a52) [Swtich pg_* extensions to use schema_post_process](https://github.com/jeremyevans/sequel/commit/2f57a52701d92bd791a7cf885cd9817d36ac4a96)


</details>